README.md

```diff
@@ -2,25 +2,46 @@ easy-license-creator
 ====================
 [![Build Status](https://travis-ci.org/DANS-KNAW/easy-license-creator.png?branch=master)](https://travis-ci.org/DANS-KNAW/easy-license-creator)
 
-<Remove this comment and extend the descriptions below>
-
+Create a license (pdf file) for a given dataset.
 
 SYNOPSIS
 --------
 
-    easy-license-creator params
+    easy-license-creator <datasetID> <template-dir> <license-file>
 
 
 DESCRIPTION
 -----------
 
-<Replace with a longer description of this module>
+A command line tool to create a license for a given dataset as a pdf file. The tool searches for a dataset that corresponds to the given `datasetID` and uses the metadata of this dataset, as well as the personal data of the depositor to generate the license.
```

Comment:
[Tang constructie](https://nl.wikipedia.org/wiki/Tangconstructie): 'license ... as' perhaps:

A command line tool that creates a pdf document containing the license for a given dataset.